scripts/g.extension/g.extension.py

```diff
@@ -396,7 +396,16 @@ def _get_addons_list(self):
                         )
         return addons_dict
 
+    def _addon_exists(self, addon_list):
+        if not [self.addons[addon] for addon in addon_list if addon in self.addons]:
+            gs.fatal(
+                _("No AddOn named <{}> found in the repository.").format(
```

Comment:
A small note, I think we use "addon" or "Addon", not AddOn, at least in the g.extension manual.
```suggestion
                _("No addon named <{}> found in the repository.").format(
```

Reply:
Ok.